src/Command/UpdateRecipesCommand.php

```diff
@@ -0,0 +1,407 @@
+        }
+
+        $io = $this->getIO();
+        if (!$this->isIndexClean($io)) {
+            $io->write([
+                '  Cannot run <comment>recipes:update</comment>: Your git index contains uncommitted changes.',
+                '  Please commit or stash them and try again!',
+            ]);
+
+            return 1;
+        }
+
+        $packageName = $input->getArgument('package');
+        $symfonyLock = $this->flex->getLock();
+        if (!$packageName) {
+            $packageName = $this->askForPackage($io, $symfonyLock);
+
+            if (null === $packageName) {
+                $io->writeError('All packages appear to be up-to-date!');
+
+                return 0;
+            }
+        }
+
+        if (!$symfonyLock->has($packageName)) {
+            $io->writeError([
+                'Package not found inside symfony.lock. It looks like it\'s not installed?',
+                sprintf('Try running <info>composer recipes:install %s --force -v</info> to re-install the recipe.', $packageName),
+            ]);
+
+            return 1;
+        }
+
+        $packageLockData = $symfonyLock->get($packageName);
+        if (!isset($packageLockData['recipe'])) {
+            $io->writeError([
+                'It doesn\'t look like this package had a recipe when it was originally installed.',
+                'To install the latest version of the recipe, if there is one, run:',
+                sprintf('  <info>composer recipes:install %s --force -v</info>', $packageName),
+            ]);
+
+            return 1;
+        }
+
+        $recipeRef = $packageLockData['recipe']['ref'] ?? null;
+        $recipeVersion = $packageLockData['recipe']['version'] ?? null;
+        if (!$recipeRef || !$recipeVersion) {
+            $io->writeError([
+                'The version of the installed recipe was not saved into symfony.lock.',
+                'This is possible if it was installed by an old version of Symfony Flex.',
+                'Update the recipe by re-installing the latest version with:',
+                sprintf('  <info>composer recipes:install %s --force -v</info>', $packageName),
+            ]);
+
+            return 1;
+        }
+
+        $originalRecipe = $this->getRecipe($packageName, $recipeRef, $recipeVersion);
+
+        if (null === $originalRecipe) {
+            $io->writeError([
+                'The original recipe version you have installed could not be found, it may be too old.',
+                'Update the recipe by re-installing the latest version with:',
+                sprintf('  <info>composer recipes:install %s --force -v</info>', $packageName),
+            ]);
+
+            return 1;
+        }
+
+        $newRecipe = $this->getRecipe($packageName);
+
+        if ($newRecipe->getRef() === $originalRecipe->getRef()) {
+            $io->write(sprintf('This recipe for <info>%s</info> is already at the latest version.', $packageName));
+
+            return 0;
+        }
+
```

Comment:
```suggestion

    $io->write(sprintf('  Updating recipe for <info>%s</info>...', $packageName));

```

The first message that appears when running a happy-flow is "Yes, the recipe is updated". While this gives you a "yay" feeling, it also feels like the common "I'm going to do stuff for you" initial message was missing.

<details>
<summary>Besides, happy-flow output also didn't mention the package name I was updating once in the output</summary>

```
   Yes! Recipe updated!


  Run git status the changes then commit like normal.

  * #916 - Routing: don't require to install doctrine/annotations to use attributes (@dunglas)
      https://github.com/symfony/recipes/pull/916
```

</detail>